eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Deleg.hs

```diff
@@ -68,7 +68,6 @@ data ConwayDelegPredFailure era
   | StakeKeyNotRegisteredDELEG !(Credential 'Staking (EraCrypto era))
   | StakeKeyHasNonZeroRewardAccountBalanceDELEG !Coin
   | DRepAlreadyRegisteredForStakeKeyDELEG !(Credential 'Staking (EraCrypto era))
-  | WrongCertificateTypeDELEG
```

Comment:
Just to make sure I understand: is it the case that the way we validate makes it impossible for this Predicate failure to occur? (But it's still possible in Shelley, since we kept it there, right?)

Reply:
That is correct. We structured the code in Conway in a smarter way that no longer requires this case. In Shelley it is also an impossible case, so we could do the same for shelley, but that'll be some time later in the future.